src/Pages/Signature/Detail/Benefit.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import {
+  Headline3,
+  Subtitle1,
+  Button,
+  Icon,
+  ButtonColor,
+  Colors,
+  ModalBottomSheet,
+} from "@class101/ui";
+import "./Benefit.scss";
+
+function Benefit(props) {
+  return (
+    <div className="benefit" ref={(el) => (props.focusTarget.current[4] = el)}>
+      <div className="benTitle">
+        <Headline3>
+          지금 적용되고 있는
+          <br />
+          혜택이에요!
+        </Headline3>
+      </div>
+      <div className="couponBox">
+        <div className={`coupon ${props.isCouponClicked ? "disabled" : ""}`}>
+          <Subtitle1>20,000원 할인 쿠폰 증정</Subtitle1>
+          <div className="coupInf">
+            <div>[시그니처] 💌 특별 할인 20,000원 쿠폰 💌</div>
+            <div>12월 31일까지 사용 가능</div>
+            <Button
+              leftIcon={props.isCouponClicked ? null : <Icon.Download />}
```

Comment:
```suggestion
              leftIcon={isCouponClicked && <Icon.Download />}
```
1. 함수 파라미터 자리에서 직접 구조분해 할당해주세요
2. true 만 활용할 경우 && 연산자 사용해주세요

Reply:
수정 완료